pkg/cmd/synonyms/save/save.go

```diff
@@ -0,0 +1,101 @@
+package save
+
+import (
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/algolia/algoliasearch-client-go/v3/algolia/opt"
+	"github.com/spf13/cobra"
+
+	shared "github.com/algolia/cli/pkg/cmd/synonyms/shared"
+	"github.com/algolia/cli/pkg/cmdutil"
+)
+
+// NewSaveCmd creates and returns a save command for index synonyms
+func NewSaveCmd(f *cmdutil.Factory, runF func(*shared.SaveOptions) error) *cobra.Command {
+	opts := &shared.SaveOptions{
```

Comment:
The `SaveOptions` should be in the same file (as its not shared with the other commands)